.github/resources-crds/buildconfig-crd.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
can you create a new sub folder and put the resources there, organize it like thus: 

```
.github/resources/
├── crds
│   ├── buildconfig-crd.yaml
│   ├── deploymentconfig-crd.yaml
│   ├── imagestream-crd.yaml
│   └── servicemonitor-crd.yaml
├── kfdef
│   └── kfdef.yaml
├── kind
│   └── kind.yaml
└── olm
    ├── catalogsource.yaml
    ├── operatorgroup.yaml
    └── subscription.yaml

```

Reply:
also in the workflow refer to the `.github/resources` via an env var, and don't hard code it, so something like `.github/resources-kfdef/` should become `${{ env.resources }}/kfdef/`

Reply:
Done, updated to have this directory structure, and refer to the .github/resources via an env var.